signer/aws/aws.go

```diff
@@ -22,54 +24,85 @@ import (
 	v4 "github.com/aws/aws-sdk-go/aws/signer/v4"
 )
 
-// OpenSearchService AWS OpenSearchService Name
+// OpenSearchService Amazon OpenSearch Service Name
```

Comment:
You could keep it as "Amazon OpenSearch Service" and for L30 as "Amazon OpenSearch Serverless".

Reply:
Also:

1. Do we _want_ to export these constants? They aren't exported in the v2 signer.
2. If we do perhaps they should be `OpenSearchServiceName` and `OpenSearchServerlessName` to clarify they are `Name`s.